domapping/cli.py

```diff
@@ -81,14 +89,23 @@ def schema_to_mapping_cli(schema, output, config, indent):
 @click.argument('output', type=click.File('w'), default='-')
 @click.option('--indent', '-i', default=4, type=click.INT,
               help='Output template indentation step.')
-@click.option('--type', '-t', default='type', type=click.STRING,
+@click.option('--type', '-t', type=click.STRING,
               help='Root type name used in jinja block names.')
 def mapping_to_jinja_cli(mapping, output, indent, type):
     """Generate jinja template from Elasticsearch mapping."""
     parsed_mapping = json.load(mapping)
 
-    template = mapping_to_jinja(parsed_mapping, type, indent=indent)
-    output.write(template)
+    if 'mappings' in parsed_mapping:
+        result = []
+        for item in parsed_mapping:
+            template = mapping_to_jinja(parsed_mapping, item, indent=indent)
+            result.append(template)
+        output.write(','.join(result))
```

Comment:
When I use the JSON in the test `test_multiple_mapping_to_jinja`:
```
{
    "mappings": {
        "test1": {
            "properties": {
                "sub1": {
                    "type1": "string"
                }
            }
        },
        "test2": {
            "properties": {
                "sub2": {
                    "type2": "string"
                }
            }
        }
    }
}
```
The output of `domapping mapping_to_jinja ...` looks like:
```
{
    {% block mappings %}
    "mappings": {"test2": {"properties": {"sub2": {"type2": "string"}}}, "test1": {"properties": {"sub1": {"type1": "string"}}}}
    {% endblock %}
}
```

IMHO it would be better if it looked more similar to what you get without a `mappings` root, i.e:
```
"mappings": {
    "test1": {
        {% block test1 %}
        "properties": {
            {% block test1__PROPERTIES__ %}
            "sub1": {
                {% block test1__sub1 %}
                "type1": "string"
                {% endblock %}
            }
            {% endblock %}
        }
        {% endblock %}
    },
    "test2": {
        {% block test2 %}
        "properties": {
            {% block test2__PROPERTIES__ %}
            "sub2": {
                {% block test2__sub2 %}
                "type2": "string"
                {% endblock %}
            }
            {% endblock %}
        }
        {% endblock %}
    }
}
```